src/tmxrasterizer/main.cpp

```diff
@@ -126,5 +129,14 @@ int main(int argc, char *argv[])
         }
     }
 
+    if (parser.isSet(QLatin1String("advance-animations"))) {
+      bool ok;
+      w.setAdvanceAnimations(parser.value(QLatin1String("advance-animations")).toDouble(&ok));
+      if (!ok || w.advanceAnimations() <= 0.0) {
```

Comment:
```suggestion
      if (!ok || w.advanceAnimations() < 0) {
```
Suggesting this change because I don't think it's useful to consider passing 0 an error, and the comparison should not be to a double.